src/tokens/fonts.js

```diff
@@ -0,0 +1,33 @@
+const fonts = {
+  family: {
+    lato: 'Lato',
+  },
+  sizes: {
+    xxs: '10px',
+    xs: '12px',
+    s: '13px',
+    m: '14px',
+    l: '16px',
+    xl: '18px',
+    xxl: '20px',
+    xxxl: '24px',
+    xxxxl: '32px',
+  },
+  weight: {
+    light: '300',
+    regular: '400',
+    bold: '700',
+  },
+  lineHeight: {
+    s: 1.2,
```

Comment:
We should use ```px``` since this will be consumed in styled-components
```suggestion
    s: '1.2px',
```

Reply:
line height never have unit associated with them

Reply:
RN throws a warning that it needs a unit. I just tried it out